rules/integrations/aws/ml_cloudtrail_error_message_spike.toml

```diff
@@ -23,41 +23,77 @@ interval = "15m"
+    - If the source is an EC2 IP address, is it associated with an EC2 instance in one of your accounts, or could
+    it be sourcing from an EC2 instance that's not under your control?
+    - If it is an authorized EC2 instance, is the activity associated with normal behavior for the instance role or roles?
+    Are there any other alerts or signs of suspicious activity involving this instance?
+- Consider the time of day. If the user is a human, not a program or script, did the activity take place during a normal
+time of day?
+- Contact the account owner and confirm whether they are aware of this activity if suspicious.
+- If the account is suspected to have been compromised, track access to servers, services, and data accessed by the
+account in the last 24 hours to scope potentially compromised assets.
 
-#### Possible investigation steps:
-- Examine the history of the error. Has it manifested before? If the error, which is visible in the `aws.cloudtrail.error_message` field, only manifested recently, it might be related to recent changes in an automation module or script.
-- Examine the request parameters. These may provide indications as to the nature of the task being performed when the error occurred. Is the error related to unsuccessful attempts to enumerate or access objects, data, or secrets? If so, this can sometimes be a byproduct of discovery, privilege escalation or lateral movement attempts.
-- Consider the user as identified by the `user.name field`. Is this activity part of an expected workflow for the user context? Examine the user identity in the `aws.cloudtrail.user_identity.arn` field and the access key ID in the `aws.cloudtrail.user_identity.access_key_id` field, which can help identify the precise user context. The user agent details in the `user_agent.original` field may also indicate what kind of a client made the request.
-- Consider the source IP address and geolocation for the calling user who issued the command. Do they look normal for the calling user? If the source is an EC2 IP address, is it associated with an EC2 instance in one of your accounts, or could it be sourcing from an EC2 instance that's not under your control? If it is an authorized EC2 instance, is the activity associated with normal behavior for the instance role or roles? Are there any other alerts or signs of suspicious activity involving this instance?
+### False positive analysis
 
-### False Positive Analysis
-- This rule has the possibility to produce false positives based on unexpected activity occurring such as bugs or recent
-changes to automation modules or scripting.
+- Examine the history of the command. If the command, which is visible in the `event.action field`, only manifested
+recently, it might be part of a new automation module or script. If it has a consistent cadence (for example, if it
+appears in small numbers on a weekly or monthly cadence), it might be part of a housekeeping or maintenance process.
```

Comment:
```suggestion
appears in small numbers on a weekly or monthly cadence), it might be part of a housekeeping or maintenance process. You can find the command in the `event.action field` field.
```